Marlin/src/lcd/extui_malyan_lcd.cpp

```diff
@@ -180,32 +183,18 @@ void process_lcd_eb_command(const char* command) {
  * X, Y, Z, A (extruder)
  */
 void process_lcd_j_command(const char* command) {
-  static bool steppers_enabled = false;
-  char axis = command[0];
-
-  switch (axis) {
-    case 'E':
-      // enable or disable steppers
-      // switch to relative
-      queue.enqueue_now_P(PSTR("G91"));
-      queue.enqueue_now_P(steppers_enabled ? PSTR("M18") : PSTR("M17"));
-      steppers_enabled = !steppers_enabled;
-      break;
-    case 'A':
-      axis = 'E';
-      // fallthru
-    case 'Y':
-    case 'Z':
-    case 'X': {
-      // G0 <AXIS><distance>
-      // The M200 class UI seems to send movement in .1mm values.
-      char cmd[20], pos[6];
-      sprintf_P(cmd, PSTR("G1 %c%s"), axis, dtostrf(atof(command + 1) / 10.0, -5, 3, pos));
-      queue.enqueue_one_now(cmd);
-    } break;
-    default:
-      SERIAL_ECHOLNPAIR("UNKNOWN J COMMAND", command);
-      return;
+  auto move_axis = [](const auto axis) {
+    const float dist = atof(command + 1) / 10.0;
+    ExtUI::setAxisPosition_mm(ExtUI::getAxisPosition_mm(axis) + dist, axis);
```

Comment:
In the most recent code, I'm back to getting this error:
`ork/exec /Users/jcnelson/Library/Arduino15/packages/STM32/tools/arm-none-eabi-gcc/8.2.1-1.7/bin/arm-none-eabi-g++: resource temporarily unavailable
/var/folders/2w/45wss0310nbfs7dbvg2mv01c0000gn/T/arduino_build_338636/sketch/src/lcd/extui_malyan_lcd.cpp: In lambda function:
/var/folders/2w/45wss0310nbfs7dbvg2mv01c0000gn/T/arduino_build_338636/sketch/src/lcd/extui_malyan_lcd.cpp:187:29: error: 'command' is not captured
     const float dist = atof(command + 1) / 10.0;`